src/lib/ngsi/Request.cpp

```diff
@@ -75,6 +75,7 @@ const char* requestType(RequestType rt)
   case Ngsi10SubscriptionsConvOp:                   return "Ngsi10SubscriptionsConvOp";
 
   case LogRequest:                                  return "Log";
+  case LogLevelRequest:                             return "LogLevel";
   case VersionRequest:                              return "Version";
```

Comment:
Sure, why not.
The name LogRequest comes from the beginning when we had debug and verbose apart from trace levels
LogTraceRequest is a better name, the way things are now